laguinho/models/metadata.py

```diff
@@ -0,0 +1,21 @@
+from marshmallow import Schema, fields, pprint, validate
+
+def get_required_error(attribute):
+        return { "required": { "message": attribute + " is required" } }
+        
+class DatasetMetadataSchema(Schema):
+    name = fields.Str(
+        required=True, 
+        error_messages=get_required_error("name"))
+    url = fields.Str(
+        required=True, 
+        error_messages=get_required_error("url"))
+    path = fields.Str(
+        required=True, 
+        error_messages=get_required_error("path"))
+    maintainers = fields.List(fields.Str())
+    format = fields.Str(
+        validate=validate.OneOf(["json", "csv"]), 
+        required=True, 
+        error_messages=get_required_error("format"))
+    contributable = fields.Bool()
```

Comment:
Poderia inicializar o schema logo aqui depois da classe:
```python
dataset_metadata = DatasetMetadataSchema()
```
pra não precisar ficar instanciando quando for usar, é OK ter o schema como um singleton